indico/modules/events/registration/views.py

```diff
@@ -24,7 +24,18 @@
 
 class WPManageRegistration(WPJinjaMixin, WPConferenceModifBase):
     template_prefix = 'events/registration/'
-    sidemenu_option = 'registration'
+
+    @property
+    def sidemenu_option(self):
+        if self._conf.getType() != 'conference':
```

Comment:
No, i think the initial one is fine. Especially since this one would not show any menu item as active if you have a participation form and then convert the event to conference